### DISCUSSION TITLE
How to take FS snapshot

### DISCUSSION THREAD

**Discussion by Yaswani341** [Opster/opensearch-k8s-operator#837] [Q&A]
How can i take FS backup in a multiple pod environment?
To take FS backup, we need to have pvc which is common for all nodes. Adding pvc under each pod component is not helping in getting common pvc.
Do i miss anything or if this capability is not present at the moment then is it something that can be implemented in the operator?

**Comment by swoehrl-mw** [comment]
Hi @Yaswani341. Currently it is not possible to mount extra PVCs on pods. There is an open issue to track this (#484).
What is already possible is mounting volumes via CSI.
Otherwise the recommended solution is to use external object storage like S3 for your snapshots.
